### ISSUE TITLE
[A11ySev2] [Screen Reader-Dashboards with Grafana – Azure Insights Key Vaults-Explore]: Screen reader does not announce the service label for the combo box under the Explore tab

### ISSUE THREAD

**Issue by LinglingTong** [grafana/grafana#118770]
## Steps to reproduce

Repro Steps: 
  
1. Open 'Azure Monitor dashboards with Grafana' in Azure Portal
2. "Monitor | Dashboards with Grafana | Gallery" screen appears with already Dashboards with Grafana tab selected.  
3. Select already existing data as 'Azure Insights Key Vaults'  under All Dashboards tab.  
4. Navigate to the  Explore  list item present in the New panel more action menu under row title. 
5. Navigate all controls present in Explore Pane. 
6. Observe that the screen reader does not announce a label for the combo box, making it unclear what the combo box is for. 

**Actual Result:**
The screen reader does not announce any label for the service combo box under the Explore tab, leaving users unaware of its purpose.  
**Observation**:  
The service combo box does not have an accessible name or label. This results in the screen reader failing to announce its purpose, which can confuse users relying on assistive technologies.  **Expected Result: ** 
  The service combo box should have a programmatically associated label that clearly describes its purpose. Screen reader should announce the 'service' label for the combo box.

**Relevant WCAG Criteria:** [4.1.2 Name, Role, Value](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html) (Level A)

## Environment

- **Grafana version:** 12.2.0
- **Data source type & version:**
- **User OS & Browser:** Version 25H2 (OS Build 26200.7093) / Chrome Version 142.0.7444.177 (Official Build) (64-bit)
- **Others:**

**Comment by LinglingTong** [comment]
<img width="1919" height="1096" alt="Image" src="https://github.com/user-attachments/assets/329347da-e8c0-481e-aa6c-838c1dc702f6" />

**Comment by ashharrison90** [comment]
dupe of https://github.com/grafana/grafana/issues/118772

**Comment by v-kavyagupta** [comment]
We verified this issue on the URL:http://localhost:3000/d/adq4djt/new-dashboard?orgId=1&from=now-7d&to=now&timezone=browser&var-adhoc0=&var-query0=&dtab=new-tab&editPanel=3
Issus is still repro,  Screen reader does not announce the service label for the combo box under the Explore tab

<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/fda20b42-8168-49da-bd65-a6aac00c1561" />
